### ISSUE TITLE
Scalable Hyperparameter Transfer Learning

### ISSUE THREAD

**Issue by sile** [sile/readings#22]
URL
-----

https://papers.nips.cc/paper/7917-scalable-hyperparameter-transfer-learning

Abstract
----------

> Bayesian optimization (BO) is a model-based approach for gradient-free black-box function optimization, such as hyperparameter optimization. Typically, BO relies on conventional Gaussian process (GP) regression, whose algorithmic complexity is cubic in the number of evaluations. As a result, GP-based BO cannot leverage large numbers of past function evaluations, for example, to warm-start related BO runs. We propose a multi-task adaptive Bayesian linear regression model for transfer learning in BO, whose complexity is linear in the function evaluations: one Bayesian linear regression model is associated to each black-box function optimization problem (or task), while transfer learning is achieved by coupling the models through a shared deep neural net. Experiments show that the neural net learns a representation suitable for warm-starting the black-box optimization problems and that BO runs can be accelerated when the target black-box function (e.g., validation loss) is learned together with other related signals (e.g., training loss). The proposed method was found to be at least one order of magnitude faster that methods recently published in the literature.

Table of Contents
--------------------

- (1) Introduction
- (2) Conclusions

BibTeX
------

```
@incollection{NIPS2018_7917,
title = {Scalable Hyperparameter Transfer Learning},
author = {Perrone, Valerio and Jenatton, Rodolphe and Seeger, Matthias W and Archambeau, Cedric},
booktitle = {Advances in Neural Information Processing Systems 31},
editor = {S. Bengio and H. Wallach and H. Larochelle and K. Grauman and N. Cesa-Bianchi and R. Garnett},
pages = {6845--6855},
year = {2018},
publisher = {Curran Associates, Inc.},
url = {http://papers.nips.cc/paper/7917-scalable-hyperparameter-transfer-learning.pdf}
}
```

See Also
----------

- #21
